packages-ts/gauntlet-terra-contracts/src/lib/contracts.ts

```diff
@@ -2,6 +2,13 @@ import { io, logger } from '@chainlink/gauntlet-core/dist/utils'
 import { JSONSchemaType } from 'ajv'
 import { existsSync, readFileSync } from 'fs'
 import path from 'path'
+import fetch from 'node-fetch'
+
+const { Octokit } = require('@octokit/core')
+const octokit = new Octokit()
+const http = require('http')
+
+export const RELEASE_VERSION = 'v0.0.4'
```

Comment:
User should be capable of injecting a version, and falling to a default one if none was provided